axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -525,6 +589,26 @@
           "address": "0xD7C77eEA2aF147D351f189eF9eC17EfDf609a3c8",
           "implementation": "0x4F1ccCE088244d2979128b3af64cD6C8D440bbC0",
           "deployer": "0xba76c6980428A0b10CFC5d8ccb61949677A61233"
+        },
+        "InterchainTokenService": {
+          "salt": "ITS v2.0.1",
```

Comment:
Flow ITS wasn't deployed with the correct salts, missing ` devnet-amplifier2` suffix